x/launch/client/cli/tx_request_remove_validator.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/spn/x/launch/types"
+)
+
+func CmdRequestRemoveValidator() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "request-remove-validator [chainID] [validatorAddress]",
+		Short: "Request to remove a genesis validator",
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
```suggestion
		Args:  cobra.MinimumNArgs(2),
```

Reply:
https://github.com/tendermint/spn/pull/220/files#r683780516

Reply:
I haven't seen it was about minimum arg👍